src/components/SearchForm/SearchForm.js

```diff
@@ -29,12 +44,13 @@ const SearchForm = ({ setArticles, setSearchQuestion, setIsLoading }) => {
       <form className="form__field" onSubmit={handleSearch}>
         <input
           type="text"
-          name="search"
+          name="question"
           placeholder="Enter topic"
           minLength="2"
           maxLength="40"
           required
           className="form__field-topic"
+          onChange={handleChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components